### ISSUE TITLE
Problem with model training

### ISSUE THREAD

**Issue by cxx226** [yixchen/YouRefIt_ERU#1]
Thank you for your excellent work!
 I tried to reproduce the performance in the paper, but the results were poor. I trained the model directly using a single GPU by

> python train.py --data_root ./ln_data/ --dataset yourefit --gpu gpu_id  --use_paf --use_sal --large

and the evaluation results are:

> Accuracy = 0.18784972022382093
> Small Accuracy = 0.0 in 418.0 samples
> Medium Accuracy = 0.005063291139240506 in 395.0 samples
> Large Accuracy = 0.5319634703196348 in 438.0 samples
> Accuracy = 0.07993605115907274
> Small Accuracy = 0.0 in 418.0 samples
> Medium Accuracy = 0.0 in 395.0 samples
> Large Accuracy = 0.228310502283105 in 438.0 samples
> Accuracy = 0.019184652278177457
> Small Accuracy = 0.0 in 418.0 samples
> Medium Accuracy = 0.0 in 395.0 samples
> Large Accuracy = 0.0547945205479452 in 438.0 samples

I'd like to know if there are any other settings needed?

**Comment by yixchen** [comment]
Hi, 

The results may vary but I think it's not converging in this case. Did you make any changes to the training epochs, batch size, or learning rate? The training log may also help diagnose the potential problems. Another way to test your setup is to train the model with only images input.
